README.md

```diff
@@ -83,7 +83,7 @@ The results here-below were computed on January the 30th, 2024 with the followin
 | tapestry     | 5.8.3    |
 | underscore   | 1.97     | 
 | yasson       | 3.0.3    |
-| wast         | 0.0.12.1 |
+| wast         | 0.0.13.2 |
```

Comment:
Please revert this change in the readme as this list of versions is meant to indicate what lib versions were tested on the charts on this page.

Reply:
This library, which has just been added, has not been tested yet, and there is no performance data for this library in the chart. It seems that it can be updated.

Reply:
Fair.